Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -393,17 +395,24 @@ public void driveWithPathFollowerInit(PathPlannerTrajectory trajectory, boolean
       resetOdometry(m_posePathStart);
       m_isFieldRelative = false;
     }
+    else  // Adds a check for the current estimated robot position
+      resetOdometry(m_poseEstimator.getEstimatedPosition( ));
```

Comment:
Let's review this. If we're going to reset our pose to the initial pose in the path, that's the first case. If we aren't, then we're using the existing robot pose--which "is what it is". I think we're resetting the odometry with the current position of the odometry.